app/views/content/is-teaching-right-for-me/engineers-teach-physics/_article.html.erb

```diff
@@ -115,16 +115,14 @@
 
   <section class="col col-content-max col-space-s">
     <%= render TextBlockWithOneImageComponent.new(
-      heading: "Support with funding",
+      heading: "Fund your teaching training",
```

Comment:
@caitlin-reidy we don't have anything on qualifications on this page. I wasn't sure if that's because you need to have a degree to be an engineer?

Reply:
I think we should probably rejig the ETP page to be more in line with the others - albeit with an additional section on the actual ETP programme as this is important

Would suggest doing this under a separate ticket so as not to delay this PR further though - I'll put one in the backlog and then let's discuss